gamemode/gamemoderun.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "gamemoderun",
+  "buildsystem": "simple",
+  "build-commands": [
+    "mv data/gamemoderun /app/bin/gamemoderun",
```

Comment:
This should be built from source.

Reply:
Oh its just a shell script.

You could simplify this by only downloading a single file.

Reply:
It doesn't need to be, as it's a shell script. I tried using the `install` command for this, but couldn't quite figure it out, so I fell back to `mv` and `chmod`.